internal/cli/deployments/options/deployment_opts.go

```diff
@@ -130,8 +131,23 @@ func (opts *DeploymentOpts) LocalMongodHostname() string {
 
 var LocalDevImage = "docker.io/mongodb/mongodb-atlas-local"
 
+func getLocalDevImage() string {
+	// First check environment variable
+	if envImage := os.Getenv("ATLAS_LOCAL_DEPLOYMENT_IMAGE"); envImage != "" {
```

Comment:
if you use the standard prefix `MONGODB_ATLAS` as in `MONGODB_ATLAS_LOCAL_DEPLOYMENT_IMAGE` then the config can automatically do the checks to read from env or the config file

Reply:
and I strongly think even if not relating to the features int he config you should follo wthe standard prefix to avoid confusion

Reply:
agree with using the standard prefix https://github.com/mongodb/mongodb-atlas-cli/blob/2224a16135637ec08ddb23f5db6375abe5623270/internal/config/profile.go#L42

Reply:
Will address the changes in a future PR

Reply:
we don't really read variables in the profile because we "cheat" by relying on viper, check https://github.com/spf13/viper?tab=readme-ov-file#working-with-environment-variables, the ones you listed are more the exception because they usually come from external tools and we use them for telemetry